pkg/controller/rest/verifiable/operation.go

```diff
@@ -37,6 +37,7 @@ const (
 	GetCredentialByNamePath    = verifiableCredentialPath + "/name" + "/{name}"
 	GetCredentialsPath         = VerifiableOperationID + "/credentials"
 	SignCredentialsPath        = VerifiableOperationID + "/signcredential"
+	deriveCredentialPath       = VerifiableOperationID + "/derivecredential"
```

Comment:
Any reason the rest are exported but not this one?

Reply:
rest are exported since those are used as endpoints in mobile binding. This one is not yet available in mobile binding. But still exported it to match others.

Reply:
fixed.